internal/campaigns/run_steps.go

```diff
@@ -157,6 +182,70 @@ func runSteps(ctx context.Context, wc *WorkspaceCreator, repo *graphql.Repositor
 	return diffOut, err
 }
 
+func parseStepRun(stepCtx StepContext, run string) (*template.Template, error) {
+	t := template.New("step-run").Delims("${{", "}}")
+
+	funcMap := template.FuncMap{
+		"repository_name":    func() string { return stepCtx.Repository.Name },
+		"search_results":     stepCtx.Repository.FileMatchPathList,
+		"search_result_list": stepCtx.Repository.FileMatchPathList,
+		"join":               func(sl []string, sep string) string { return strings.Join(sl, sep) },
+	}
+
+	funcMap["modified_files"] = stepCtx.PreviousStep.ModifiedFiles
```

Comment:
Just curious as to why this line is separate from the rest of the items in the FuncMap.

Reply:
Because the `PROTOTYPE` in the title implies "hacky shit, don't expect anything to make sense" 😄 It's a leftover.